HW3_new/README.md

```diff
@@ -0,0 +1,30 @@
+# React + TypeScript + Vite
+
```

Comment:
the instructions here is not very clear and couldnt build the project. also i dont know what to install to build this.

Reply:
Updated